pkg/ffuf/job.go

```diff
@@ -175,25 +194,33 @@ func (j *Job) runProgress(wg *sync.WaitGroup) {
 	defer wg.Done()
 	totalProgress := j.Input.Total()
 	for j.Counter <= totalProgress {
+
 		if !j.Running {
 			break
 		}
+
 		j.updateProgress()
 		if j.Counter == totalProgress {
 			return
 		}
+
+		if !j.RunningJob {
+			return
+		}
+
 		time.Sleep(time.Millisecond * time.Duration(j.Config.ProgressFrequency))
 	}
 }
 
 func (j *Job) updateProgress() {
 	prog := Progress{
-		StartedAt:  j.startTime,
-		ReqCount:   j.Counter,
-		ReqTotal:   j.Input.Total(),
-		QueuePos:   j.queuepos,
-		QueueTotal: len(j.queuejobs),
-		ErrorCount: j.ErrorCounter,
+		StartedAt:        j.startTimeJob,
+		StartedAtProcess: j.startTime,
```

Comment:
This isn't currently used anywhere in the progress output, so I think it should be removed.

Reply:
Same here, removed StartedAtProcess.